README.md

```diff
@@ -1,5 +1,40 @@
-This repository backs Jenkins jobs on an internal Continuous Infrastructure Jenkins instance: https://atomic-e2e-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/
+# aos-cd-jobs
 
-Multibranch jobs:
-- Branch prefix: build/*   : https://atomic-e2e-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/aos-cd-builds/
-- Branch prefix: cluster/* : https://atomic-e2e-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/aos-cd/ 
+This repository backs Jenkins jobs on a couple of Jenkins masters.
+
+## Jenkins pipeline definitions under `jobs/`
+ 
+An internal [Continuous Infrastructure Jenkins instance](https://atomic-e2e-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/) indexes Jenkinsfiles in the branches of this repository, which are generated from the Jenkinsfiles that live under the `jobs/` directory. Jobs under the `jobs/build/` directory are indexed at the [`aos-cd-builds`](https://atomic-e2e-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/aos-cd-builds/) grouping, while the jobs under `jobs/cluster/` are indexed at the general [`aos-cd`](https://atomic-e2e-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/aos-cd/) grouping. A quick synopsis of these indexed jobs is as follows:
```

Comment:
This is a great start. We can add now or later a description of how /Jenkinsfile in master does the splitting whenever the repo changes due to this job: https://atomic-e2e-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/aos-cd-master/ . It polls for changes in the repo and does the branch splitting when they are detected.